test/specs/scale.time.tests.js

```diff
@@ -123,7 +124,8 @@ describe('Time scale tests', function() {
 
 			var scaleOptions = Chart.scaleService.getScaleDefaults('time');
 			var scale = createScale(mockData, scaleOptions);
-			scale.update(1000, 200);
+			scale.chart.width = 1100;
+			scale.update(1100, 200);
```

Comment:
Why are we changing the `width` in so many of these tests?

Creating the scale and then changing the width and calling `update` makes it harder to step through the code with a debugger because the scale gets created and then updated so the first time you hit any line of code it doesn't count. Can we pass the width into `createScale` instead so it's created with the correct width from the beginning?